src/box/internal.tsx

```diff
@@ -43,7 +44,8 @@ export default function InternalBox({
     styles[`color-${color || 'default'}`],
     styles[`font-size-${fontSize || 'default'}`],
     styles[`font-weight-${fontWeight || 'default'}`],
-    styles[`t-${textAlign}`]
+    styles[`t-${textAlign}`],
+    isRefresh && styles.refresh
```

Comment:
Is this one necessary? I don't see any `.refresh` selector in the stylesheets of this component.

Reply:
We do no longer need these changes in box component. I originally added to differentiate the color from h1 to h5 variant in box component, but I figured out that it's not necessary. 

I'll remove them and push the change.